src/car_controller/src/car_controller.py

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env python
+#from ..world.nodes import car
```

Comment:
@asalani93 Do you know how to include files that are up a couple of directories?